src/bootstrap/src/utils/change_tracker.rs

```diff
@@ -101,4 +101,9 @@ pub const CONFIG_CHANGE_HISTORY: &[ChangeInfo] = &[
         severity: ChangeSeverity::Warning,
         summary: "rust-analyzer-proc-macro-srv is no longer enabled by default. To build it, you must either enable it in the configuration or explicitly invoke it with x.py.",
     },
+    ChangeInfo {
+        change_id: 119373,
+        severity: ChangeSeverity::Warning,
+        summary: "Removed dist.missing-tools config options, as it has been unused for some time.",
+    },
```

Comment:
The `Warning` level should be used when we remove config options, right?

> /// Used when the default value of an option changes, or support for an option is removed entirely,
> /// potentially requiring developers to update their build configurations.

I'm a bit confused on why does it differ if we remove the config option now vs in the middle of 2024. The config option hasn't been used for some time anyway. Do you think that in 6 months less people will have it in their configs? :thinking:

Reply:
> The Warning level should be used when we remove config options, right?

Actually right. I got confused on this field because as it's not a functional one

Reply:
> Do you think that in 6 months less people will have it in their configs?

That's the problem no one can know -  That's why I think it's worth to keep it for a while.

Reply:
I just wonder how does it differ e.g. from https://github.com/rust-lang/rust/blob/1f141dc0b8635f17ce31d72e3328ad2b0dd6b084/src/bootstrap/src/utils/change_tracker.rs#L94 - here we removed two options (immediately after we found that they are unused), and it was `Info` :laughing:

Reply:
Those we didn't have in config.example.toml

Reply:
Ah, I see, good point. So, if we're not removing this option (yet), should we even add any change entry? I guess that the only actionable item for bootstrap users could be something like "Remove this option if you use it, it will disappear soon.".

Reply:
> So, if we're not removing this option (yet), should we even add any change entry?

I think we should add an info entry similar to "x is deprecated as it was an unused option; remove it if you using it, it will disappear soon"

Reply:
Ok, done.